ramls/inventory-reports.raml

```diff
@@ -18,9 +19,16 @@ resourceTypes:
   collection-get: !include raml-util/rtypes/collection-get.raml
 
 /inventory-reports:
-    displayName: In transit items report
+    displayName: Item-based reports
     /items-in-transit:
         type:
           collection-get:
             exampleCollection: !include examples/items-in-transit.json
-            schemaCollection: items
+            schemaCollection: items-in-transit
+
+    /pick-slips:
```

Comment:
Are pick slips a report (from the POs perspective)? 

Are the reasons for changing pick slips the same as for the other reports? 

Would a module want to definitely implement these together? 

Would the same client want to consume them?

I’m asking these questions because pick slips seem different to me to the other reports, and maybe warrant their own separate interface?

Reply:
You are right, it is different from other reports we have. I changed the name to just `PickSlips` and the endpoint to `/circulation/pick-slips`. How does that sound?

Reply:
I think that is preferable to it being a report. I think we might consider alternative paths like /circulation/slips/pick but that is a minor concern.

I think the most important thing is that this endpoint should be in a separate interface to the rest of the `circulation` interface. And be marked experimental. 

This helps with interface dependency management and allows it to evolve easier in the early development (as I still have concerns about the API design, with respect to templates and context)

Reply:
I note that you have already done that, so I shall resolve this comment